frontend.sh

```diff
@@ -85,11 +85,32 @@ clean_and_install() {
 # Run tasks to build the project for distribution.
 build() {
   echo 'Building project…'
-  gulp clean
-  gulp build
+
+  if [ cfgov/static_built/css/main.css -ot cfgov/unprocessed/css/ ]; then
```

Comment:
it would be nice if this could use though config file, I know it's a fantasy though.